qutebrowser/browser/commands.py

```diff
@@ -451,9 +451,10 @@ def tab_take(self, index, keep=False):
 
     @cmdutils.register(instance='command-dispatcher', scope='window')
     @cmdutils.argument('win_id', completion=miscmodels.window)
+    @cmdutils.argument('private', flag='p')
     @cmdutils.argument('count', value=cmdutils.Value.count)
     def tab_give(self, win_id: int = None, keep: bool = False,
-                 count: int = None) -> None:
+                 count: int = None, private=False) -> None:
```

Comment:
Needs a type annotation:

```suggestion
                 count: int = None, private: bool = False) -> None:
```